modules/core/src/main/scala/net/message/ParameterDescription.scala

```diff
@@ -16,8 +14,6 @@ object ParameterDescription {
   final val Tag = 't'
 
   val decoder: Decoder[ParameterDescription] =
-    int16.flatMap { n =>
-      int32.asDecoder.replicateA(n).map(ParameterDescription(_))
-    }
+    codecs.listOfN(int16, int32).map(ParameterDescription(_))
```

Comment:
The StackOverflowError was being thrown by this line. So the same solution you provided in #148, just different place ;)